packages/server/src/automations/automationUtils.ts

```diff
@@ -163,7 +163,7 @@ async function generateAttachmentRow(attachment: AutomationAttachment) {
 
   try {
     const { filename } = attachment
-    const extension = path.extname(filename)
+    const extension = path.extname(filename).replaceAll(".", "")
```

Comment:
Is this expected? Do we want to trim? What will happen with complex extensions? `fe, .tar.gz`